src/watertap_contrib/seto/costing/units/med_tvc_surrogate.py

```diff
@@ -0,0 +1,218 @@
+import pyomo.environ as pyo
```

Comment:
At some point we will want a header for these files. Won't let it hold up the PR.